### DISCUSSION TITLE
Skin editor can be opened during gameplay

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#30952] [Q&A]
### Type

Game behaviour

### Bug description

Pressing the shortcut for the skin editor during live gameplay will cause the skin editor to open, and make it impossible to play at all without first closing the skin editor, which will likely result in a fail.

This does not have to be during a break.

### Version

2024.1115.3-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17985167/compressed-logs.zip)

**Comment by peppy** [comment]
This is intentional right now, but we'll reconsider if people manage to hit this regularly.

Have you changed your skin editor binding? The default should be impossible to accidentally hit during gameplay.

**Reply by JacksonChen666** [reply]
> Have you changed your skin editor binding? The default should be impossible to accidentally hit during gameplay.

My quick retry keybind is Alt-Shift (that's it), my skin editor keybind is Ctrl-Shift-S (default IIRC), and my tapping keys are A and S. I'm now not exactly sure how I pressed the skin editor keybind, but I probably did press quick retry and mispressed it into Ctrl-Shift-S

**Reply by utauyo** [reply]
> The default should be impossible to accidentally hit during gameplay.  

I am currently using D and F as my keybinds, and just now, while doing some _crazy ass streams_, my hand tilted left and managed to hit that shortcut - resulting in a fail.  
I guess my ring finger hit S and my pinky finger mashed both Ctrl and Shift

**Reply by peppy** [reply]
You can unbind it from settings to fix this, but thanks for providing the insight.
